src/main/java/games/strategy/triplea/delegate/InitializationDelegate.java

```diff
@@ -80,23 +80,17 @@ public boolean delegateCurrentlyRequiresUserInput() {
     return false;
   }
 
-  protected void init(final IDelegateBridge aBridge) {
-    initDestroyerArtillery(aBridge);
-    initShipyards(aBridge);
-    initTwoHitBattleship(aBridge);
-    initOriginalOwner(aBridge);
-    initTech(aBridge);
-    initSkipUnusedBids(aBridge.getData());
-    initAiStartingBonusIncome(aBridge);
-    initDeleteAssetsOfDisabledPlayers(aBridge);
-    initTransportedLandUnits(aBridge);
-    resetUnitState(aBridge);
-  }
+  protected void init(final IDelegateBridge delegateBridge) {
+    initDestroyerArtillery(delegateBridge);
+    initShipyards(delegateBridge);
+    initTwoHitBattleship(delegateBridge);
+    initOriginalOwner(delegateBridge);
+    initTech(delegateBridge);
+    initSkipUnusedBids(delegateBridge.getData());
+    initAiStartingBonusIncome(delegateBridge);
+    initDeleteAssetsOfDisabledPlayers(delegateBridge);
+    initTransportedLandUnits(delegateBridge);
 
-  /**
-   * The initTransportedLandUnits has some side effects, and we need to reset unit state to get rid of them.
-   */
-  private void resetUnitState(final IDelegateBridge aBridge) {
```

Comment:
Consistency is a bit debatable, resetUnitState is resolving side effects from the other init methods. The method blocks are a decent grouping, but a series of void method calls do not really help readability. There is no benefit beyond having comment blocks.

With that said, the javadoc comment here is really useful,  removing it does lose context.